cmd/createGitlab.go

```diff
@@ -214,6 +214,13 @@ var createGitlabCmd = &cobra.Command{
 
 			informUser("Waiting for Gitlab dns to propagate before continuing", globalFlags.SilentMode)
 			gitlab.AwaitHost("gitlab", globalFlags.DryRun)
+
+			domain := viper.GetString("aws.hostedzonename")
+			isValidCertificate, err := pkg.AwaitValidLetsEncryptCertificateNTimes(domain, globalFlags.DryRun, 60)
+			if err != nil || !isValidCertificate {
+				log.Println(err)
```

Comment:
just valid certificate would be enough. We don't need the full `AwaitValidLetsEncryptCertificateNTimes`

Reply:
The issue is for valid certificates in general.

Reply:
do you mean, one single request to see if the certificate is invalid?

Reply:
I refer to the long name of the function, details on the next comment, the issue is not about let's encrypt it is about certs that are trusted.